CONTRIBUTING.md

```diff
@@ -152,7 +151,7 @@ Follow the following instructions to start contributing.
   **2.** Clone your forked copy of the project.
 
   ```
-  git clone https://github.com/layer5io/layer5.git
+  git clone https://github.com/<your-github-username>/layer5.git
```

Comment:
This change is incorrect.

Reply:
doing `git clone https://github.com/layer5io/layer5.git` would just clone the original repo and **not the fork**